Makefile

```diff
@@ -139,6 +142,9 @@ fmt: install-tools
 fmt-sh: install-tools
 	${SHFMT} -w -d -i 5 .
 
+linter: install-tools
```

Comment:
can we change `linter` to `lint` everywhere since it should describe the job not the tool

Reply:
Make sense. Fixed it in the newest commit.